### ISSUE TITLE
update plotting documentation to reflect new range option and text box positioning (Trac #621)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#753]
Lewis has fixed some long standing plotting requests so that the plotting range can be given directly and also that text on the plot once created can still be moved around.  These new features need to be added to the documentation.

Migrated from http://trac.sasview.org/ticket/621
```json
{
    "status": "closed",
    "changetime": "2016-09-06T05:58:03",
    "_ts": "2016-09-06 05:58:03.657522+00:00",
    "description": "Lewis has fixed some long standing plotting requests so that the plotting range can be given directly and also that text on the plot once created can still be moved around.  These new features need to be added to the documentation.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-08-16T21:32:03",
    "component": "SasView",
    "summary": "update plotting documentation to reflect new range option and text box positioning",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.0.0",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/09/05 14:52:26`: 
* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "accepted"

**Comment by butlerpd** [comment]
Trac update at `2016/09/06 05:58:03`: 
* **butler** commented:

In changeset f9b0c81e56a680c4cdbca2d3251ba73fc4b903cd:

```text
#!CommitTicketReference repository="" revision="f9b0c81e56a680c4cdbca2d3251ba73fc4b903cd"
update the graph/plotting documentation with new set axis range option,
fixed information on modifying plots and added new image for linearized
fits that includes new look.  Fixes #753
```


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "accepted" to "closed"
